netbox/extras/plugins/__init__.py

```diff
@@ -70,9 +71,14 @@ def ready(self):
             register_template_extensions(template_extensions)
 
         # Register navigation menu items (if defined)
+        try:
+            menu_header = import_object(f"{self.__module__}.{self.menu_header}")
+        except AttributeError:
+            menu_header = None
```

Comment:
This doesn't need to be wrapped in a `try` block: `import_object()` will return None if the attribute doesn't exist.